scio-extra/src/main/scala/com/spotify/scio/extra/bigquery/AvroConverters.scala

```diff
@@ -28,6 +28,10 @@ import scala.jdk.CollectionConverters._
 object AvroConverters extends ToTableRow with ToTableSchema {
 
   @experimental
+  @deprecated(
+    "use saveAsBigQueryTable from the com.spotify.scio.bigquery package instead",
```

Comment:
should we add that it will be removed in 0.12.0? (i think that's what we agreed on...?)